server/routerlicious/packages/routerlicious-base/src/test/alfred/api.spec.ts

```diff
@@ -216,19 +216,19 @@ describe("Routerlicious", () => {
 
 				describe("/api/v1", () => {
 					it("/ping", async () => {
-						await assertThrottle("/api/v1/ping", null, null);
+						await assertThrottle("/apis/ping", null, null);
```

Comment:
I don't know for sure how these tests work, but if they run a local build of the website for this, then our redirects won't be applied. That _could_ be why these tests are failing. It's probably okay to leave these tests pointing at the v1 docs.